tests/integration/targets/cloudtrail/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# Beware: most of our tests here are run in parallel.
+# To add new tests you'll need to add a new host to the inventory and a matching
+# '{{ inventory_hostname }}'.yml file in roles/s3_bucket/tasks/
+
```

Comment:
```suggestion
```

Looks like this was copy and paste from a set of "parallel" integration tests :)